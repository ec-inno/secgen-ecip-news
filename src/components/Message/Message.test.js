import React from 'react';
import renderer from 'react-test-renderer';
import Message from './Message';

describe('Message', () => {
  it('renders correctly', () => {
    const props = {
      close: null,
      variant: 'warning',
      icon: {
        shape: 'notifications--warning',
        size: 'l',
      },
      title: 'Disclaimer',
      description:
        'The contents on this page are the sole responsibility of the organisers of the initiatives. The texts reflect solely the views of their authors and can in no way be taken to reflect the views of the European Commission.',
    };
    const tree = renderer.create(<Message {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
