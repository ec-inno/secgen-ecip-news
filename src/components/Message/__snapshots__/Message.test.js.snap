// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message renders correctly 1`] = `
<div
  className="ecl-message ecl-message--warning"
  data-ecl-message={true}
  role="alert"
>
  <svg
    aria-hidden="true"
    className="ecl-message__icon ecl-icon ecl-icon--l"
    focusable="false"
  >
    
    
    <use
      xlinkHref="file#notifications--warning"
    />
  </svg>
  <div
    className="ecl-message__content"
  >
    
    <div
      className="ecl-message__title"
    >
      Disclaimer
    </div>
    <p
      className="ecl-message__description"
    >
      The contents on this page are the sole responsibility of the organisers of the initiatives. The texts reflect solely the views of their authors and can in no way be taken to reflect the views of the European Commission.
    </p>
  </div>
</div>
`;
